roles/kubernetes-apps/container_runtimes/meta/main.yml

```diff
@@ -20,3 +20,10 @@ dependencies:
       - apps
       - crun
       - container-runtimes
+
+  - role: kubernetes-apps/container_runtimes/youki
```

Comment:
this should probably be conditioned by a supported container manager, as far as I can tell from the project's public documentation they only support `cri-o`.